README.md

```diff
@@ -10,24 +10,12 @@
   </strong>
 </p>
 
-<p align="center">
```

Comment:
I think the original code looked nicer because it is center aligned

Reply:
I agree, and I kinda hate this reference syntax anyway...